### ISSUE TITLE
LD -XLinker is invalid option compiling for native app

### ISSUE THREAD

**Issue by zgagnon** [Tensegritics/ClojureDart#281]
**Describe the bug**

Building the example app fails with error:

```ld: unknown option: -Xlinker```

Does your problem persist after `clj -M:cljd clean && flutter clean`?

Yes

**To Reproduce**
Follow quick start steps. Launch in macos app with:

```clj -M:cljd flutter -d macos -v ```

**Expected behavior**
Macos native app opens

**Additional context**

- System: Macos Sonoma, apple silicon
- Flutter --version:
```
flutter --version
Flutter 3.13.8 • channel stable • https://github.com/flutter/flutter.git
Framework • revision 6c4930c4ac (7 days ago) • 2023-10-18 10:57:55 -0500
Engine • revision 767d8c75e8
Tools • Dart 3.1.4 • DevTools 2.25.0
```
- Cjojure --version:
```
clojure --version
Clojure CLI version 1.11.1.1413
```
- Cocoapods version: Version: 1.13.0

**Comment by dupuchba** [comment]
hello @zgagnon , what example app are you talking about?
The quick example or on of the samples?

**Comment by zgagnon** [comment]
The quick example from the readme. Flutter doctor is fine, and running a dart app works in this environment.

**Comment by cgrand** [comment]
@zgagnon what's your MacOS version? And Xcode?
Anyway it's a Flutter issue. Looks a bit like https://github.com/flutter/flutter/issues/137250 where someone points to 
<img width="943" alt="image" src="https://github.com/Tensegritics/ClojureDart/assets/47025/20d423d0-914d-4962-84b4-1cc33803819a">

Have you enabled rosetta? Is your Mac OS a fresh install or an old install migrated from intel?

**Comment by zgagnon** [comment]
This may be something else with Flutter (above didn't work). I did, however do an XCode upgrade, and now all Flutter MacOS connection is not building, not just when launching with ClojureDart. It seems the latest version of XCode does not work with Flutter, and previous versions are not supported on MacOS Sonoma. 

I've opened an issue with Flutter about this (https://github.com/flutter/flutter/issues/137391). I believe I'm unable to develop flutter apps on this machine at the moment. Feel free to close this issue, or leave it open for tracking, as best suits you
